src/PIL/TiffTags.py

```diff
@@ -179,7 +179,7 @@ def lookup(tag):
     33432: ("Copyright", ASCII, 1),
     34377: ("PhotoshopInfo", BYTE, 1),
     # FIXME add more tags here
-    34665: ("ExifIFD", SHORT, 1),
+    34665: ("ExifIFD", LONG, 1),
     34675: ("ICCProfile", UNDEFINED, 1),
     34853: ("GPSInfoIFD", BYTE, 1),
```

Comment:
This one should also be `LONG`

Reply:
This will be part of #3980 though, as per [your comment](https://github.com/python-pillow/Pillow/issues/3973#issuecomment-511944566), yes?

Reply:
Just to note here - the scope of #3980 has been reduced, and changing the type of GPSInfoIFD is now in #4009